client/my-sites/plans/current-plan/trials/business-trial-included.tsx

```diff
@@ -9,21 +9,26 @@ import useBusinessTrialIncludedFeatures from './use-business-trial-included-feat
 interface Props {
 	translate: typeof translate;
 	displayAll: boolean;
+	displayOnlyActionableItems?: boolean;
 }
 const BusinessTrialIncluded: FunctionComponent< Props > = ( props ) => {
-	const { displayAll = true } = props;
+	const { displayAll = true, displayOnlyActionableItems = false } = props;
 
 	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) ) || -1;
 	const siteSlug = useSelector( ( state ) => getSelectedSiteSlug( state ) ) || '';
 	const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
 
 	const allIncludedFeatures = useBusinessTrialIncludedFeatures( siteSlug, siteAdminUrl || '' );
 
-	const whatsIncluded = displayAll
+	let whatsIncluded = displayAll
 		? allIncludedFeatures
 		: // Show only first 4 items
 		  allIncludedFeatures.slice( 0, 4 );
 
+	if ( displayOnlyActionableItems ) {
+		whatsIncluded = whatsIncluded.filter( ( item ) => item.buttonClick );
+	}
```

Comment:
This is because we are displaying the "Hosting included" card in the post-checkout trial screen. It's a feature, not an action, so I'm including the easiest way to remove the card.